### DISCUSSION TITLE
getAccountAPTAmount return value

### DISCUSSION THREAD

**Discussion by bjmvercelli** [aptos-labs/aptos-developer-discussions#193] [Questions] [answered]
### Discord user ID

361196456922841091

### Describe your question in detail.

Hey! I'm starting to learn aptos development and came up with this situation.

As you can see, i'm executing a getAccountAPTAmount and it is returning **100000**, what's the reason of that? For amounts > 1 it returns "correctly"...

![image](https://github.com/aptos-labs/aptos-developer-discussions/assets/61213222/b46b5cce-ab17-45a8-aa88-7b4edb035738)

### What have you tried or looked at?  Or how can we reproduce the error?

Yes

### Which operating system are you using?

Linux (Ubuntu, Fedora, Windows WSL, etc.)

### Which SDK or tool are you using? (if any)

TypeScript SDK

**Comment by 0x-j** [comment, marked as answer]
it returns in octas which is smallest unit of APT where 1 APT = 10^8 octas

**Reply by bjmvercelli** [reply]
Thanks! 

Is it the same for other functions? For example, if i execute `transferCoinTransaction()` the `amount` prop should be passed in octa?

**Reply by 0x-j** [reply]
Correct, but please always verify with a small amount or on testnet first. If not then there's a bug
